inst/sympref.m

```diff
@@ -424,3 +439,12 @@
 
 %!error <invalid preference or command> sympref ('nosuchsetting')
 %!error <invalid preference or command> sympref ('nosuchsetting', true)
+
+%!test
+%! syms x
```

Comment:
probably don't need `syms x`.  Can you move this test up before the `r = sympref('reset')` test?  Around line 415 please.